tests/Benchmarks/System.Binary.Base64/Base64TestHelper.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is it necessary to duplicate this file? Would including a reference to System.Binary.Base64.Tests.csproj in Benchmarks.csproj be sufficient? We should try to avoid such code duplication.